components/expenses/ExpenseAttachedFiles.js

```diff
@@ -20,7 +29,7 @@ const ExpenseAttachedFiles = ({ files, onRemove, showInvoice, collective, expens
               <UploadedFilePreview
                 onClick={downloadInvoice}
                 isDownloading={isLoading}
-                fileName={getExpenseInvoiceFilename(collective, expense)}
+                fileName={<FormattedMessage id="File.DownloadExpense" defaultMessage="Download expense" />}
```

Comment:
This prop expects a string, not a React component. This generates a warning in the console and produces an unexpected result when hovered:

![image](https://user-images.githubusercontent.com/1556356/162741842-320a9f17-cdaf-4e91-940c-29ed20fe91a8.png)